.github/workflows/migration-check.yml

```
@@ -0,0 +1,44 @@
+
+on:
+  pull_request:
+    paths:
+      - 'backend/**/migrations/*.py'
+
+jobs:
+  migration-check:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)